cmake/OpenCVConfig.cmake

```diff
@@ -82,6 +82,8 @@ if(MSVC)
     set(OpenCV_RUNTIME vc11)
   elseif(MSVC_VERSION EQUAL 1800)
     set(OpenCV_RUNTIME vc12)
+  elseif(MSVC_VERSION EQUAL 1900)
+    set(OpenCV_RUNTIME vc14)
```

Comment:
They skipped that version number, it doesn't exist :-)

vc14 is Visual Studio 2015 and has MSVC_VERSION 1900.

I think their initial idea was to have vc14 and Visual Studio 2014 (so there wouldn't be any discrepancy anymore between both numbers), but since it's 2015 now and VS2014 got renamed to VS2015, that discrepancy still exists.